src/core_modules/capture-core/components/ListView/Main/listViewMain.types.ts

```diff
@@ -0,0 +1,35 @@
+import type { ReactNode } from 'react';
+import type { WithStyles } from '@material-ui/core';
+import type { ListViewContextBuilderPassOnProps } from '../ContextBuilder';
+import type {
+    Columns,
+    CustomMenuContents,
+    CustomRowMenuContents,
+    CustomTopBarActions,
+    SetColumnOrder,
+    SelectRow,
+} from '../types';
+
+type WithFilterPassOnProps = ListViewContextBuilderPassOnProps & {
+    filters: ReactNode;
+};
+
+type ComponentProps = {
+    columns: Columns;
+    filters: ReactNode;
+    updatingWithDialog?: boolean;
+    onSetColumnOrder: SetColumnOrder;
+    rowIdKey: string;
+    customMenuContents?: CustomMenuContents;
+    customRowMenuContents?: CustomRowMenuContents;
+    customTopBarActions?: CustomTopBarActions;
+    onClickListRow: SelectRow;
+    onRowSelect: (id: string) => void;
+    onSelectAll: (rows: Array<string>) => void;
+    isSelectionInProgress?: boolean;
+    bulkActionBarComponent: ReactNode;
+} & WithStyles<any>;
```

Comment:
Move the WithStyles from here to src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.tsx on line 41 :
Props & WithStyles<typeof getStyles>